specification/columns/chargecategory.md

```diff
@@ -33,10 +30,11 @@ Allowed values:
 
 | Value      | Description                          |
 | :--------- | :------------------------------------|
-| Adjustment | Any adjustments that are applied after the original usage or purchase row. Adjustments may be related to multiple charges.   |
+| Usage      | Charges based on the quantity of a service or resource that was consumed over a given period of time.     |
 | Purchase   | Charges for the acquisition of a service or resource bought upfront or on a recurring basis.              |
 | Tax        | Applicable taxes that are levied by the relevant authorities. Tax charges may vary depending on factors such as the location, jurisdiction, and local or federal regulations. |
-| Usage      | Charges based on the quantity of a service or resource that was consumed over a given period of time.     |
+| Adjustment      | Positive or negative charges the provider applies that do not fall into other category values.    |
```

Comment:
This implies that an adjustment is only a non-zero value, so let's explicit about that.

```suggestion
| Adjustment      | Non-zero charges the provider applies that do not fall into other category values.    |
```